Computer Vision/WEAPON-DETECTION-SYSTEM/weapon_detection.py

```diff
@@ -0,0 +1,74 @@
+import cv2
+import numpy as np
+
+# Load Yolo
+net = cv2.dnn.readNet("yolov3_training_2000.weights", "yolov3_testing.cfg")
+net.setPreferableBackend(cv2.dnn.DNN_BACKEND_DEFAULT)
+net.setPreferableTarget(cv2.dnn.DNN_TARGET_CPU)
+classes = ["Weapon"]
+
+# Open the video file
+cap = cv2.VideoCapture("pistol1.mp4")
```

Comment:
Also, include some videos that include pistol and some without it to see the accuracy and performance of the model. Have at least 3 videos in the folder.